docs/static/js/load-fullstory.js

```diff
@@ -0,0 +1,21 @@
+window['_fs_host'] = 'fullstory.com';
+window['_fs_script'] = 'edge.fullstory.com/s/fs.js';
+window['_fs_org'] = 'o-1RQ4W2-na1';
```

Comment:
guessing this can be shared as this is analytics

Reply:
Yes public id.